From 8a3da979b73b6927b4c6adcdbf6fc3bf4abe1a78 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Fri, 5 Sep 2014 01:31:39 -0400
Subject: [iommu] vt-d: Mark function eoi_ioapic_pin_remapped() as static in irq_remappi

Message-id: <20140905013139.30867.94378.stgit@gir.stowe>
Patchwork-id: 91269
O-Subject: [RHEL7.1 PATCH 05/28] iommu/vt-d: Mark function eoi_ioapic_pin_remapped() as static in irq_remappi
Bugzilla: 1129808
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1129808
Upstream Status: d2d1e8fe430c5d26781b14695b05ba46ae8f525b

commit d2d1e8fe430c5d26781b14695b05ba46ae8f525b
Author: Rashika Kheria <rashika.kheria@gmail.com>
Date:   Wed Dec 18 12:06:45 2013 +0530

    iommu/vt-d: Mark function eoi_ioapic_pin_remapped() as static in irq_remappi

    Mark function eoi_ioapic_pin_remapped() as static in irq_remapping.c
    because it is not used outside this file.

    This eliminates the following warning in
    irq_remapping.c:drivers/iommu/irq_remapping.c:153:6: warning: no
    previous prototype for 'eoi_ioapic_pin_remapped' [-Wmissing-prototypes]

    Signed-off-by: Rashika Kheria <rashika.kheria@gmail.com>
    Reviewed-by: Josh Triplett <josh@joshtriplett.org>
    Signed-off-by: Joerg Roedel <joro@8bytes.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/iommu/irq_remapping.c b/drivers/iommu/irq_remapping.c
index 39f81ae..3b05d1b 100644
--- a/drivers/iommu/irq_remapping.c
+++ b/drivers/iommu/irq_remapping.c
@@ -150,7 +150,7 @@ static int irq_remapping_setup_msi_irqs(struct pci_dev *dev,
   return do_setup_msix_irqs(dev, nvec);
 }
 
-void eoi_ioapic_pin_remapped(int apic, int pin, int vector)
+static void eoi_ioapic_pin_remapped(int apic, int pin, int vector)
 {
  /*
   * Intr-remapping uses pin number as the virtual vector
-- 
1.7.1